packages/base/src/observer.ts

```diff
@@ -0,0 +1,45 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import {PropertyValues} from '@polymer/lit-element/lib/updating-element.js';
+
+export interface Observer {
+  (value: any, old: any): void;
```

Comment:
can this be `unknown`? to ensure where it is used, proper type guards are used etc

Reply:
Hmm, I get type errors trying to define the value type in the `switch` observer if these are `unknown`.

```
packages/switch/src/mwc-switch.ts:33:13 - error TS2345: Argument of type '(this: Switch, value: boolean) => void' is not assignable to parameter of type 'Observer'.
  Types of parameters 'value' and 'value' are incompatible.
    Type 'unknown' is not assignable to type 'boolean'.

33   @observer(function(this: Switch, value: boolean) {
               ~~~~~~~~
```